egs/librimix/ConvTasNet/local/prepare_data.sh

```diff
@@ -6,17 +6,14 @@ python_path=python
 
 . ./utils/parse_options.sh
 
-current_dir=$(pwd)
-# Clone LibriMix repo
-git clone https://github.com/JorisCos/LibriMix
-
-# Run generation script
-cd LibriMix
-. generate_librimix.sh $storage_dir
-
-cd $current_dir
-$python_path local/create_local_metadata.py --librimix_dir $storage_dir/Libri$n_src"Mix"
+if [[ $n_src -le  1 ]]
+then
+  changed_n_src=2
+else
+  changed_n_src=n_src
+fi
```

Comment:
Can you explain that please?

Reply:
This is because Libri1Mix doesn't really exist it's a part of Libri2Mix. To access the right directory we do `Libri $n_src Mix` because of that the recipes with `n_src=1` doesn't work we have to change it to 2 which a bit awkward

Reply:
Ok, thanks.